mmv1/templates/terraform/encoders/spanner_instance_config_update.go.erb

```diff
@@ -0,0 +1,16 @@
+project, err := tpgresource.GetProject(d, meta.(*transport_tpg.Config))
+if err != nil {
+return nil, err
+}
+obj["name"] = fmt.Sprintf("projects/%s/instances/%s", project, obj["name"])
+newObj := make(map[string]interface{})
+newObj["instanceConfig"] = obj
+updateMask := make([]string, 0)
+if d.HasChange("display_name") {
+  updateMask = append(updateMask, "displayName")
+}
+if d.HasChange("labels") {
+  updateMask = append(updateMask, "labels")
+}
+newObj["fieldMask"] = strings.Join(updateMask, ",")
```

Comment:
It looks like instanceConfigs supports standard `updateMask`: https://cloud.google.com/spanner/docs/reference/rest/v1/projects.instanceConfigs/patch - would it be possible to use that instead? Not well documented but: https://github.com/GoogleCloudPlatform/magic-modules/blob/a1848839cb9855f70e935106ab3ad03d66efedb3/mmv1/api/resource.rb#L50